string name
int16 data01
int16 data02
int16 data03
int16 data04